relay-filter/src/common.rs

```diff
@@ -51,7 +54,7 @@ pub enum FilterStatKey {
 
 impl FilterStatKey {
     /// Returns the string identifier of the filter stat key.
-    pub fn name(self) -> &'static str {
+    pub fn name(self) -> String {
```

Comment:
Save one allocation for non-generic filters:

```suggestion
    pub fn name(self) -> Cow<'static, str> {
```